src/routes/transactions/entities/schemas/propose-transaction.dto.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { JSONSchemaType } from 'ajv';
+import { ProposeTransactionDto } from '../propose-transaction.dto.entity';
+
+export const proposeTransactionDtoSchema: JSONSchemaType<ProposeTransactionDto> =
+  {
+    $id: 'https://safe-client.safe.global/schemas/transactions/propose-transaction.dto.json',
+    type: 'object',
+    properties: {
+      to: { type: 'string' },
+      value: { type: 'string' },
+      data: { type: 'string', nullable: true },
+      nonce: { type: 'string' },
+      operation: { type: 'number', enum: [0, 1] },
+      safeTxGas: { type: 'string' },
+      baseGas: { type: 'string' },
+      gasPrice: { type: 'string' },
+      gasToken: { type: 'string', nullable: true },
```

Comment:
`gasToken` should be required and non-nullable

Reply:
Fixed in [0b60e2e](https://github.com/safe-global/safe-client-gateway-nest/pull/292/commits/0b60e2e077ddd4ba2cec258d3903a52f5141487d)